com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -50,36 +49,35 @@ internal class InferenceAnalytics
         const int k_MaxNumberOfElements = 1000;
 
 
+#if UNITY_EDITOR && MLA_UNITY_ANALYTICS_MODULE_ENABLED
```

Comment:
updates to analytics conditional compiling after writing a test for EnableAnalytics()